docs/docs/configuration/object_detectors.md

```diff
@@ -291,3 +291,52 @@ To verify that the integration is working correctly, start Frigate and observe t
 
 
 # Community Supported Detectors
+
+## Rockchip RKNN-Toolkit-Lite2
+This detector is only available if one of the following Rockchip SoCs is used:
+- RK3566/RK3568
+- RK3588/RK3588S
+- RV1103/RV1106
+- RK3562
+
+These SoCs come with a NPU that will highly speed up detection.
+
+### Setup
+
+All you have to do is to use the `frigate:latest-rk` docker image. Here is a minimal `docker-compose.yml`:
+```
+version: "3.9"
+services:
+  frigate:
+    container_name: frigate
+    image: frigate:latest-rk
+    privileged: true
+    restart: unless-stopped
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ./data/config:/config
+      - ./data/media:/media/frigate
+    ports:
+      - "5000:5000"
+```
```

Comment:
I think we should avoid showing the docker compose here, the installation docs are used for this already. This would create a higher chance that changes to the docker compose file in the future are forgotten to be reflected here

Reply:
Yes, maybe I will just write, that docker must run in privileged mode.

Reply:
Yeah makes sense